cli/src/main/java/org/jboss/as/cli/handlers/ifelse/ExpressionParser.java

```diff
@@ -138,6 +138,8 @@ protected Operand parseOperand() throws CommandLineException {
                     } else {
                         --depthCount;
                     }
+                } else if (input.charAt(endIndex) == '"') { // traverse over string operands surrounded by quotes
```

Comment:
If we introduce escaping here then any character could be escaped including the character checked in this loop, which means proceeding handling '(', ')' or '"' we have to make sure there is no '\' (or an uneven number of '\') in front.